integreat_cms/static/src/js/pages/machine-translation-overlay.ts

```diff
@@ -0,0 +1,267 @@
+/* eslint no-magic-numbers: ["error", { "ignore": [0, 1, 3] }] */
+import { getCsrfToken } from "../utils/csrf-token";
+
+const limitOfPreview = 5;
+
+const toggleMachineTranslationForPagesButton = () => {
+    // Only activate button if budget is enough
+    const budget = document.getElementById("machine-translation-overlay-budget-remains-result")
+        .innerText as any as number;
+    const MachineTranslationButton = document.getElementById(
+        "machine-translation-overlay-bulk-action-execute"
+    ) as HTMLButtonElement;
+    if (budget >= 0 && document.getElementById("machine-translation-overlay-pages").children.length > 0) {
+        MachineTranslationButton.disabled = false;
+    } else {
+        MachineTranslationButton.disabled = true;
+    }
+};
+
+const getSelectedItems = () => {
+    // get the HTML tr element of the selected pages
+    const checkboxes = document.querySelectorAll(".bulk-select-item");
```

Comment:
```suggestion
    const checkboxes = document.querySelectorAll(".bulk-select-item:checked");
```
You can directly retrieve the selected checkboxes.